pkg/noun/vortex.c

```diff
@@ -50,7 +59,8 @@ u3v_boot(u3_noun eve)
       return c3n;
     }
 
-    u3A->roc = u3k(u3t(pro));
+    u3A->roc   = u3k(u3t(pro));
+    u3A->eve_d = len_d;
```

Comment:
Just for my own clarity, what are these changes in `u3v_boot` for?

Reply:
Calculate the length of the lifecycle sequence, store that length and result globally. "boot and persist the result". This function was old and unused prior to this PR, the in-use equivalent is `_serf_play_life()` in `serf.c` (which also includes IPC response details).